webservices/legal_docs/index_management.py

```diff
@@ -333,8 +334,11 @@
 def initialize_legal_docs():
     """
     Initialize Elasticsearch for storing legal documents.
-    Create the `docs` index, and set up the aliases `docs_index` and `docs_search`
-    to point to the `docs` index. If the `doc` index already exists, delete it.
+    Create the `docs` and `archived_murs` index.
+    Set up the alias `docs_index` to point to the `docs` index only.
```

Comment:
Set up the alias `archived_murs_index` to point to the `archived_murs` index only